lib/Signaling/BackendNotifier.php

```diff
@@ -336,7 +337,8 @@ public function participantsModified(Room $room, array $sessionIds): void {
 				'participantPermissions' => Attendee::PERMISSIONS_CUSTOM,
 				'displayName' => $attendee->getDisplayName(),
 			];
-			if ($attendee->getActorType() === Attendee::ACTOR_USERS) {
+			if ($attendee->getActorType() === Attendee::ACTOR_USERS
+					|| $attendee->getActorType() === Attendee::ACTOR_FEDERATED_USERS) {
 				$data['userId'] = $attendee->getActorId();
```

Comment:
This could be problematic as there could be a conflict?
Is it required by the HPB, or could it be either a new attribute `$data['cloudId']` or dedicated actor type+id fields?

Reply:
I assumed that the `actorId` of a federated user always includes the `@remote` part (based on what I saw in my test instances) :thinking:

If they do not then 705633861094c66572678fd914239b28a1ca5b64 will need to be fixed, as that one affects how receives the signaling messages (the change here is only about the data sent and probably a different field could be used if needed).

Reply:
> I assumed that the actorId of a federated user always includes the @remote part (based on what I saw in my test instances)

They do, but it can still conflict. UserA@ServerA could also be a valid email address that they received as user ID on ServerB when being invited via the guest app.

Reply:
OK, something for a follow up then.